Formula/openblas.rb

```diff
@@ -23,6 +23,7 @@ def install
     ENV["DYNAMIC_ARCH"] = "1"
     ENV["USE_OPENMP"] = "1"
     ENV["NO_AVX512"] = "1"
+    ENV["TARGET"] = "HASWELL"
```

Comment:
This would do what homebrew uses for the bottles

```suggestion
    ENV["TARGET"] = Hardware.oldest_cpu
```

Reply:
I talked to the openBLAS people and my specific issues are apparently caused by a bug in 3.10 which will be fixed in 3.11.

But it's still important to set the TARGET because I think people can still have issues if the machine bottling the libraries has a newer processor than what they use.  So Hardware.oldest_cpu seems wise.

Reply:
You need to make sure `Hardware.oldest_cpu` is something OpenBLAS understands though.